tests/test_config.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from cleanlab_tlm.errors import TlmBadRequestError
+from cleanlab_tlm.tlm import TLM
+from cleanlab_tlm.utils.config import (
+    get_default_context_limit,
+    get_default_model,
+)
+from tests.constants import (
+    CHARACTERS_PER_TOKEN,
+)
```

Comment:
I dont see this file in the PR?

Reply:
This is an existing file. I'm following the existing test design wherein constants required for unit tests are stored in tests/constants.py